compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmValueGen.java

```diff
@@ -606,7 +585,10 @@ private byte[] createObjectValueClass(BObjectType objectType, String className,
         this.createObjectFields(cw, fields);
 
         List<BIRNode.BIRFunction> attachedFuncs = typeDef.attachedFuncs;
-        if (attachedFuncs != null) {
```

Comment:
Is it safe to remove this check? Will there be any possibilities for the `attachedFuncs` be null?

Reply:
I could not find any case `attachedFuncs` can be `null`;